### ISSUE TITLE
pip cannot retrieve license

### ISSUE THREAD

**Issue by khatkar** [plamere/spotipy#698]
**Describe the bug**
Pip cannot retrieve the actual MIT license, instead provides the name of the license file itself. This effectively prevents some  license verification utilities from correctly verifying the spotipy license.

**Your code**
`pip show spotipy`

**Expected behavior**
```
Name: spotipy
Version: 2.18.0
...
License: MIT
...
```

**Output**
```
Name: spotipy
Version: 2.18.0
...
License: LICENSE.md
...
```

**Environment:**
 - OS: Ubuntu
 - Python version: 3.8.6
 - spotipy version: 2.18.0
 - your IDE: Terminal

**Additional context**
A solution would be to rename the LICENSE.md file as LICENSE and use GitHub to generate the file as described [here](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).

**Comment by stephanebruckert** [comment]
Ok, does https://github.com/plamere/spotipy/pull/699 look good?

**Comment by stephanebruckert** [comment]
Fixed in 2.19.0
